app/assets/javascripts/controllers/report_data_controller.js

```diff
@@ -355,7 +361,6 @@
   ReportDataController.prototype.setDefaults = function() {
     this.settings.selectAllTitle = __('Select All');
     this.settings.sortedByTitle = __('Sorted By');
-    this.settings.isLoading = false;
```

Comment:
(removed because `setDefaults` is only called from the same `getData` response handler as that `setLoading` in `$timeout` above, but too early because no timeout)